File: Solar Apocalypse (Forge)/src/main/java/com/smushytaco/solar_apocalypse/Sunscreen.java
package com.smushytaco.solar_apocalypse;

import net.minecraft.world.effect.MobEffect;
import net.minecraft.world.effect.MobEffectCategory;

public class Sunscreen extends MobEffect {
    public static final Sunscreen INSTANCE = new Sunscreen();

    protected Sunscreen() {
        super(MobEffectCategory.BENEFICIAL, 14981690);
    }

    @Override
    public boolean isDurationEffectTick(int a, int b) {
        return true;
    }
}

File: Solar Apocalypse/src/main/kotlin/com/smushytaco/solar_apocalypse/Sunscreen.kt
package com.smushytaco.solar_apocalypse
import net.minecraft.world.effect.MobEffect
import net.minecraft.world.effect.MobEffectCategory
object Sunscreen: MobEffect(MobEffectCategory.BENEFICIAL, 14981690) {
    override fun shouldApplyEffectTickThisTick(duration: Int, amplifier: Int) = true
}